packages/shared/topology/sidebar/node/NodeResources.tsx

```diff
@@ -0,0 +1,164 @@
+import { resourceStatus } from '@odf/shared/status/Resource';
+import { Status } from '@odf/shared/status/Status';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import {
+  formatBytesAsMiB,
+  formatCores,
+  resourcePathFromModel,
+} from '@odf/shared/utils';
+import {
+  PrometheusResponse,
+  ResourceStatus,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+import { NodeKind, PodKind } from 'packages/shared/types';
+import './node-resources.scss';
+
+type PodOverviewItemProps = {
+  pod: PodWithMetricsKind;
+};
+
+const PodOverviewItem: React.FC<PodOverviewItemProps> = ({ pod }) => {
+  const { t } = useCustomTranslation();
+  const {
+    metadata: { name, namespace },
+    metrics: { cpu, memory },
+  } = pod;
+  const formattedCores = isNaN(cpu) ? 'N/A' : `${formatCores(cpu)} cores`;
+  const formattedMemory = isNaN(memory)
+    ? 'N/A'
+    : `${formatBytesAsMiB(memory)} MiB`;
+
+  return (
+    <li className="list-group-item container-fluid">
+      <div className="row">
+        <span className="col-xs-5">
+          <ResourceLink
+            link={resourcePathFromModel(PodModel, name, namespace)}
+            resourceModel={PodModel}
+            resourceName={name}
+          />
+        </span>
+        <span className="col-xs-3">
+          <ResourceStatus additionalClassNames="hidden-xs">
+            <Status status={resourceStatus(pod)} />
+          </ResourceStatus>
+        </span>
+        <span className="col-xs-2">
+          <span className="pod-item-column-header">{t('Memory')}</span>
+          <span className="pod-item-column-content">{formattedMemory}</span>
+        </span>
+        <span className="col-xs-2">
+          <span className="pod-item-column-header">{t('CPU')}</span>
+          <span className="pod-item-column-content">{formattedCores}</span>
+        </span>
+      </div>
+    </li>
+  );
+};
+
+type PodOverviewListProps = {
+  pods: PodWithMetricsKind[];
+};
+
+const PodsOverviewList: React.FC<PodOverviewListProps> = ({ pods }) => (
+  <ul className="list-group">
+    {_.map(pods, (pod) => (
+      <PodOverviewItem key={pod.metadata.uid} pod={pod} />
+    ))}
+  </ul>
+);
+
+type PodWithMetricsKind = PodKind & {
+  metrics: {
+    cpu: number;
+    memory: number;
+  };
+};
+
+const getPodMetric = (
+  metrics: PrometheusResponse['data']['result'],
+  podName
+): number => {
+  const podMetric = _.findIndex(
+    metrics,
+    (metric: any) => _.get(metric, 'metric.pod', '') === podName
+  );
+  return podMetric > -1 ? Number(metrics[podMetric]['value'][1]) : NaN;
+};
+
+type NodeResourcesProps = {
+  node: NodeKind;
+};
+
+const NodeResources: React.FC<NodeResourcesProps> = ({ node }) => {
+  const { t } = useCustomTranslation();
+  const [pods, loaded, loadError] = useK8sWatchResource<PodWithMetricsKind[]>({
+    kind: PodModel.kind,
+    isList: true,
+    namespaced: true,
+    namespace: 'openshift-storage',
```

Comment:
```suggestion
    namespace: CEPH_STORAGE_NAMESPACE,
```
I am not sure if I am referring to the correct constant but it's present in our constants already.

Reply:
Done.